src/platform/forms-system/src/js/components/PreSubmitSection.jsx

```diff
@@ -30,7 +30,7 @@ export function PreSubmitSection({
 }
 
 PreSubmitSection.propTypes = {
-  onChange: PropTypes.func.isRequired,
+  sectionCompleted: PropTypes.func.isRequired,
   preSubmitInfo: PropTypes.object.isRequired,
   showError: PropTypes.bool,
 };
```

Comment:
Awesome job with adding propTypes and I like the semantic change from `onChange` to `sectionCompleted`; that will be very helpful for future usage of this component.

Reply:
Thank you was suggested by @cvalarida, cant take all the credit 😉